### DISCUSSION TITLE
App icon “white”, when selected in settings, no longer displayed on the Home Screen.

### DISCUSSION THREAD

**Discussion by nicodimuscanis** [community/community#70052] [Mobile] [answered]
![image](https://github.com/community/community/assets/14107470/ebfb4aca-10ae-483d-a238-6f8037c68139)

<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.134.2 (112648498)
Device: iPhone SE (iOS 15.7.9)
</details>

**Comment by DJLink** [comment]
You can try a couple things, 

- Open Github app, go to Settings -> App Icon and set another icon, see if that works.
- Reboot of the phone (just shutdown and power on again). 
- Uninstall and reinstall the app.

Hopefully one of these will work

**Reply by nicodimuscanis** [reply]
@DJLink thanks for the reply!

Changing the icon works just fine, but it does not solve the problem. I got used to use white icon. 

I've already tried resetting my phone before and it didn't help. 

I just reinstalled the application and that didn't help either. This is what I get when I try to change the default icon
![image](https://github.com/community/community/assets/14107470/3ed3b4d4-d4a2-4308-a1b7-2cd274213a2f)

**Reply by DJLink** [reply]
ah that's a bummer. It might be iOS icon cache messed up somehow. Sorry I can't help more. 
In that case I would suggest to file a bug report with Github on this page. https://support.github.com/contact/bug-report

**Comment by nicodimuscanis** [comment, marked as answer]
The recent version 1.137.0 fixes the problem.
